### ISSUE TITLE
Overlay to add events while using another app

### ISSUE THREAD

**Issue by maffe** [simra-project/simra-android#22]
It would be nice to have the buttons to add events in front of another (navigation) app. That way one does not have to switch apps to add events.

**Comment by selphiron** [comment]
yeah, this is a good idea. If someone is up to the task, go ahead :)
